Carthage support is broken in carthage 0.15.1

Trying to install in Carthage using a Cartfile fails on carthage 0.15.1 with the message **Failed to read file or folder**.

I've opened an issue at Carthage, and the maintainers say it's an issue with IBAnimatable's plist.info , declaring an app target and not an framework target (see https://github.com/Carthage/Carthage/issues/1218#issuecomment-202058521 ).

Following are the reproduction details:

Created a folder with the following Cartfile:
`
github "JakeLin/IBAnimatable"
`

Installed Carthage 0.15.1 via homebrew.

Getting the **Failed to read file or folder** when running **carthage update**:

```
amitayd@Amitays-MacBook-Pro:~/code/carthageBug$ carthage update
*** Fetching IBAnimatable
*** Checking out IBAnimatable at "2.0"
*** xcodebuild output can be found in /var/folders/mv/n3cq2bh54q70brj2g8gpbbzh0000gn/T/carthage-xcodebuild.ON7LAQ.log
*** Building scheme "IBAnimatable" in IBAnimatableApp.xcodeproj
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/PanInteractiveAnimator.swift:27:51: warning: use '#selector' instead of explicitly constructing a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/PanInteractiveAnimator.swift:32:70: warning: use '#selector' instead of explicitly constructing a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:15:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:19:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:23:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/PanInteractiveAnimator.swift:27:51: warning: use '#selector' instead of explicitly constructing a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/PanInteractiveAnimator.swift:32:70: warning: use '#selector' instead of explicitly constructing a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:15:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:19:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:23:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/PanInteractiveAnimator.swift:27:51: warning: use '#selector' instead of explicitly constructing a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/PanInteractiveAnimator.swift:32:70: warning: use '#selector' instead of explicitly constructing a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:15:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:19:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:23:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/PanInteractiveAnimator.swift:27:51: warning: use '#selector' instead of explicitly constructing a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/PanInteractiveAnimator.swift:32:70: warning: use '#selector' instead of explicitly constructing a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:15:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:19:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
/Users/amitayd/code/carthageBug/Carthage/Checkouts/IBAnimatable/IBAnimatable/TableViewCellDesignable.swift:23:29: warning: use of string literal for Objective-C selectors is deprecated; use '#selector' or explicitly construct a 'Selector'
Failed to read file or folder at /Users/amitayd/Library/Developer/Xcode/DerivedData/IBAnimatableApp-eqqkqtewnxvrasghwofhtzfrsrar/Build/Products/Release-iphoneos/IBAnimatable.framework
```

Seems like the build itself in carthage-xcodebuild.ON7LAQ.log completes successfully:

```
...
** BUILD SUCCEEDED **
```

Reproduced with XCode 7.2 and 7.3.

Downgrading by install Carthage 0.11 from https://github.com/Carthage/Carthage/releases resolves the issue.

--- a/IBAnimatable/Info.plist
+++ b/IBAnimatable/Info.plist
@@ -1,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundleName</key>
+	<string>$(PRODUCT_NAME)</string>
+	<key>CFBundlePackageType</key>
+	<string>FMWK</string>
+	<key>CFBundleShortVersionString</key>
+	<string>2.1.0</string>
+	<key>CFBundleSignature</key>
+	<string>????</string>
+	<key>CFBundleVersion</key>
+	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>NSPrincipalClass</key>
+	<string></string>
+</dict>
+</plist>

--- a/IBAnimatableApp.xcodeproj/project.pbxproj
+++ b/IBAnimatableApp.xcodeproj/project.pbxproj
@@ -245,6 +245,7 @@
 		AEE552AF1C839EB7009CF623 /* PresenterManager.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = PresenterManager.swift; sourceTree = "<group>"; };
 		AEEE200A1C18CFD200AD033B /* NavigationBarDesginable.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = NavigationBarDesginable.swift; sourceTree = "<group>"; };
 		AEEE200C1C18D10C00AD033B /* DesignableNavigationBar.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = DesignableNavigationBar.swift; sourceTree = "<group>"; };
+		AEEF86BD1CA8E9E200899AAB /* Info.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		CAB562E75DABEED7FCE0F309 /* FillDesignable.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = FillDesignable.swift; sourceTree = "<group>"; };
 		CAB56645FCD11BEC66200818 /* SystemTransitionType.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = SystemTransitionType.swift; sourceTree = "<group>"; };
 		CAB5692C91C9ECC8018DE83E /* Navigator.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = Navigator.swift; sourceTree = "<group>"; };
@@ -423,6 +424,7 @@
 				AE0FAD3E1C1ED26A00B5289B /* segue */,
 				AE154B2A1C7BB5370093C05B /* common */,
 				E2E28E701C6A6E5E003F3852 /* scripts */,
+				AEEF86BD1CA8E9E200899AAB /* Info.plist */,
 			);
 			path = IBAnimatable;
 			sourceTree = SOURCE_ROOT;
@@ -824,7 +826,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
-				INFOPLIST_FILE = "$(SRCROOT)/IBAnimatableApp/Info.plist";
+				INFOPLIST_FILE = "$(SRCROOT)/IBAnimatable/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
@@ -845,7 +847,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
-				INFOPLIST_FILE = "$(SRCROOT)/IBAnimatableApp/Info.plist";
+				INFOPLIST_FILE = "$(SRCROOT)/IBAnimatable/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";

--- a/IBAnimatableApp/Info.plist
+++ b/IBAnimatableApp/Info.plist
@@ -17,11 +17,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>2.1.0</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>$(CURRENT_PROJECT_VERSION)</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
 	<key>UILaunchStoryboardName</key>

